playbooks/preflight-check.yml

```diff
@@ -175,3 +175,20 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1372 Check NFS exsiting
+- name: Check if nfs mount exists
+  hosts: glance_all 
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Look for NFS mount
+      shell: |
+        df -hT | grep -c 'nfs'
+      register: nfs_mount_found
+
+    - debug:
+        msg: |
+          NFS mount is found in glance container.
+          Please add NFS mount config to 'openstack-ansible-os_glance/defaults/main.yml'
+      failed_when: nfs_mount_found.stdout > 0
```

Comment:
You actually need to check for additional conditions like glance_default_store is file and nfs is the desired storage, where the latter is problematic as you only know if if a nfs mount point exists on the physical hosts where the glance container runs

Reply:
I would add checking of 'glance_default_store' on user_osa_variables_defaults.yml. Also would add check on glance_container: /etc/glance/glance-api.conf to see if 'default_store' was 'file'. But I didn't have a chance to verify rest parts of 'glance-api.conf' with 'NFS' desired setting. Could you share more details or keywords of nfs we can look for in customer environment?